### PR TITLE
Notification pop-up on connectivity problems

### PR DESCRIPTION
**Pull request by tmasternak**
## Overview
This PR handles Monitoring connectivity issues in a similar way that we handle them for SC.

The notification pop-up is shown that can be dismissed. To reconnect page refresh is needed.

### CODE REVIEW

File: src/ServicePulse.Host/app/modules/monitoring/js/services/services.monitoring.js [hunk 1]
```diff
@@ -2,7 +2,17 @@
...
+
+        var isDisconnected = false;
+
+        var notifyDisconnected = function() {
+            if (isDisconnected === false) {
+                toastService.showError("Can not connect to Monitoring");
+            }
+
+            isDisconnected = true;
+        }
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/monitoring/js/services/services.monitoring.js]: Semicolon missing here

File: src/ServicePulse.Host/app/modules/monitoring/js/services/services.monitoring.js [hunk 2]
```diff
@@ -2,7 +2,17 @@
...
+    function Service($http, rx, scConfig, uri, $q, toastService) {
+
+        var isDisconnected = false;
+
+        var notifyDisconnected = function() {
+            if (isDisconnected === false) {
+                toastService.showError("Can not connect to Monitoring");
+            }
+
+            isDisconnected = true;
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/monitoring/js/services/services.monitoring.js]: When does this ever get reset?

Comment (WilliamBZA) [reply src/ServicePulse.Host/app/modules/monitoring/js/services/services.monitoring.js]: Nevermind, I actually read the description now :P

**Comment by sergioc** [comment]
> To reconnect page refresh is needed.

In way we can make it auto reconnect when the monitoring server is available again?

**Comment by tmasternak** [comment]
> In way we can make it auto reconnect when the monitoring server is available again?

Not easy thing to do.

**Comment by tmasternak** [comment]
@sergioc reload information added (no link).

**Pull request merged**